src/modules/username.rs

```diff
@@ -4,32 +4,40 @@ use crate::configs::username::UsernameConfig;
 use crate::formatter::StringFormatter;
 use crate::utils;
 
+const ROOT_UID: Option<u32> = Some(0);
+#[cfg(not(target_os = "windows"))]
+const USERNAME_ENV_VAR: &str = "USER";
+
+#[cfg(target_os = "windows")]
+const USERNAME_ENV_VAR: &str = "USERNAME";
+
 /// Creates a module with the current user's username
 ///
 /// Will display the username if any of the following criteria are met:
 ///     - The current user isn't the same as the one that is logged in (`$LOGNAME` != `$USER`)
 ///     - The current user is root (UID = 0)
 ///     - The user is currently connected as an SSH session (`$SSH_CONNECTION`)
 pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
-    let user = context.get_env("USER");
+    let username = context.get_env(USERNAME_ENV_VAR)?;
     let logname = context.get_env("LOGNAME");
 
-    const ROOT_UID: Option<u32> = Some(0);
     let user_uid = get_uid();
 
+    let is_not_login = logname.is_some() && username != logname.unwrap_or_default();
```

Comment:
Nitpick:
I'd prefer
```suggestion
        let is_not_login = logname.map(|logname| username != logname).unwrap_or(false);
```
or
```suggestion
    let is_not_login = logname.is_some() && username != logname.unwrap();
```
here